src/applications/disability-benefits/all-claims/components/ReviewCardField.jsx

```diff
@@ -151,11 +158,13 @@ export default class ReviewCardField extends React.Component {
 
     const { volatileData, editTitle } = this.props.uiSchema['ui:options'];
     const title = editTitle || this.getTitle();
+    const subtitle = this.getSubtitle();
 
     return (
       <div className="review-card">
         <div className="review-card--body input-section va-growable-background">
           <h4 className="review-card--title">{title}</h4>
+          <div>{subtitle}</div>
```

Comment:
What do you think of making this block conditional?
```jsx
{subtitle && <div>{subtitle}</div>}
```

Reply:
Yep, agree